app/views/users/profile.html.erb

```diff
@@ -45,7 +45,7 @@
     <div class="tab-pane active" id="research">
       <div class="btn-group text-center" style="margin-bottom:10px;">
         <a href="#notes" class="btn btn-default" data-toggle="tab">Authored <%= "(#{ @notes.count })" %></a>
-        <a href="#coauthored" class="btn btn-default" data-toggle="tab">Coauthored  <%= "(#{ @coauthored.count })" %></a>
+        <a href="#co-authored" class="btn btn-default" data-toggle="tab">Co-authored  <%= "(#{ @co-authored.count })" %></a>
```

Comment:
You just need to change the Coauthored to Co-authored only at one place (Where it is displayed as text, i.e second coauthored) since `#coauthored` is a link and `@coauthored` is a global variable which is already defined.